lib/mix/lib/mix/tasks/deps.clean.ex

```diff
@@ -73,6 +73,14 @@ defmodule Mix.Tasks.Deps.Clean do
     apps -- Enum.map(deps, &Atom.to_string(&1.app))
   end
 
+  defp invalid_path_warning?(path, dependency) do
```

Comment:
Yeah I hesitated between the `?` and the `maybe`, I did not know if it was _idiomatic_ in Elixir to use `maybe`. Much cleaner this way 👍